/// Files: virt/kvm/kvm_main.c
/// Fix: f8be156be163a052a067306417cd0ff679068c97
/// Detect-To: add6a0cd1c5ba51b201e1361b05a5df817083618

virtual detect

@err exists@
identifier pfn;
position p;
@@

hva_to_pfn_remapped(...)
{
	...
*	kvm_get_pfn@p(pfn);
	...
}


@script:python depends on detect@
p << err.p;
@@

coccilib.report.print_report(p[0], 'ERROR: CVE-2021-22543')
